backend/src/main/java/de/neuefische/allyourfavorites/controller/FavoriteSoccerController.java

```diff
@@ -25,4 +24,19 @@ public FavoriteSoccerController(FavoriteSoccerService favoriteSoccerService) {
         return favoriteSoccerService.getListOfSoccerTeams();
     }
 
+    @GetMapping("favorites")
+    public List<String> getAllFavoritesOfUser(Principal principal) {
+        return favoriteSoccerService.getAllFavoritesOfUser(principal.getName());
+    }
+
+    @PostMapping
+    public String addFavorite(@RequestBody String nameOfFavorite, Principal principal) {
```

Comment:
For naming - consistency regarding the removeFavorite - method I suggest to change nameOfFavorite_ to _favoriteTeamName_
```suggestion
    public String addFavorite(@RequestBody String favoriteTeamName, Principal principal) {
```